threshold_tools_exceedance.ipynb

```diff
@@ -0,0 +1,159 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2e222069-ccfd-4139-8ad8-73e33094f70d",
+   "metadata": {},
+   "source": [
+    "# Counting threshold exceedance events\n",
+    "## Cal-Adapt Analytics Engine: threshold tools\n",
+    "\n",
+    "This notebook calculates exceedance events (time periods above or below a certain threshold) at different time scales using the `app.explore.threshold()` tool, using different criteria describing the extreme event severity and duration (for example, how many 3-day periods per year contain at least five hours of temperatures above 100 degrees F?). For any given location using hourly data, we can calculate and display how frequencies of extreme events will change under different global emissions scenarios and across model simulations. This tool can be used for applications such as:\n",
+    "- Examining how the frequency of cold snaps or heat waves of a user-defined temperature (e.g. above 95 degF), duration (e.g. 3 hours or more), and period of time (e.g. for 6 days in a row) will change annually or seasonally\n",
+    "- Exploring changes in the frequency of extreme precipitation events (for example, periods of low to no precipitation or high precipitation within a season or year) under different global emissions scenarios\n",
+    "\n",
+    "*Coming soon!* Export functionality of your selected timeseries transformations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12d3d1b9-dc2d-419f-a4dd-9ae31bbe92b1",
+   "metadata": {},
+   "source": [
+    "## Step 0: setup\n",
+    "Import the [climakitae](https://github.com/cal-adapt/climakitae) library and other dependencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f7d3d54-c3ac-4bd6-b268-dc7cd3a33e5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "pn.extension()\n",
+    "\n",
+    "import climakitae as ck\n",
+    "from climakitae import threshold_tools\n",
+    "import xarray as xr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d286f285-1eaf-4249-9a07-1a4b0f3b1d5d",
+   "metadata": {},
+   "source": [
+    "Additionally, get set up to make the computing go faster, by executing the following cell. It may take a minute or two to spin up!"
```

Comment:
This is kind of odd wording, what about instead: "Additionally, to make the computing go faster, we'll spin up a compute cluster in the following cell. Hang tight-- it may take a minute or two!"